SPECS/ca-certificates/ca-certificates.spec

```diff
@@ -324,6 +324,9 @@ rm -f %{pkidir}/tls/certs/*.{0,pem}
 %{_bindir}/bundle2pem.sh
 
 %changelog
+* Tue Jan 09 2024 Cameron Baird <cameronbaird@microsoft.com> - 3.0.0-1
+- Initial version for AzureLinux 3.0.
```

Comment:
Is there a better wording I could use here?

Reply:
I'd just check if it's "AzureLinux" or "Azure Linux"? I always though it's the latter.